eos_data_distribution/DirTools/test_class.py

```diff
@@ -1,5 +1,5 @@
 import pytest
-import DirTools
+import eos_data_distribution.DirTools
```

Comment:
This needs to be `from eos_data_distribution import DirTools`, or, you can use `from .. import DirTools`